src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -800,15 +805,48 @@ class EntityApiServiceSpec extends ApiServiceSpec {
         assertResult(StatusCodes.OK) {
           status
         }
+
+        def attrToString(a: Attribute): String = {
+          a match {
+            case AttributeString(value) => value
+            case AttributeNumber(value) => value.toString
+            case AttributeValueList(list) if list.isEmpty => ""
+            case AttributeValueList(list) => attrToString(list.head)
```

Comment:
this is the case where lists should be sorted alphabetically, not numerically, so taking the head is probably sufficient to define the correct ordering on our test list